src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball();
+    }
```

Comment:
here you should follow comment in ColorSupplier and make class variables of Random and ColorSupplier and use them to produce values you will pass to new Ball constructor